Automattic/jetpack
# Stats Widget: render from within Odyssey Stats behind a feature flag

Fixes #29795 

## Proposed changes:

- Refactored Odyssey asset loading to `CDN_Assets`
- Refactored Odyssey config data to `Odyssey_Config_Data` to share with the Stats Widget
- Render Odyssey Widget behind a feature flag `odyssey_widget`


Works with https://github.com/Automattic/wp-calypso/pull/75058

### Other information:


## Jetpack product discussion
n/a

## Does this pull request change what data or activity we track or use?
no

## Testing instructions:
* `/wp-admin/index.php`
* Ensure Stats widget works well
* Add `odyssey_widget=1` to URL
* Ensure Odyssey Widget is showing like the screenshot below.

<img width="1253" alt="image" src="https://user-images.githubusercontent.com/1425433/228455412-7a252ea3-bfc3-4641-b2bb-5e56bfbe50a6.png">
